events/wmt22.md

```diff
@@ -345,7 +345,8 @@ The results were determined with a monolingual [direct assessment](/associations
 
 #### English →
 
-The results were determined with a bilingual direct assessment in a full document, except for Livonian.
+The results were determined with a bilingual direct assessment with scalar quality metric in a full document.
```

Comment:
Let's link **direct assessment** and **scalar quality metric** to [human evaluation metrics](https://machinetranslate.org/human-evaluation-metrics) too.